android/wait_for_emulator

```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
```

Comment:
This file could instead be moved to examples/android. Now that [I now how to run the android examples on Travis](https://github.com/brasmusson/cucumber-jvm/blob/02c45303bb8825da6fb160d3da67ea6653b48c7d/.travis.yml#L10-L33) (with appropriate content in [.travis_android_update](https://github.com/brasmusson/cucumber-jvm/blob/02c45303bb8825da6fb160d3da67ea6653b48c7d/.travis_android_update)), it feels pretty interesting to do that - to avoid that the examples suddenly has stopped working.